vkrunner/vr-config.c

```diff
@@ -27,18 +27,35 @@
 
 #include "vr-config-private.h"
 
+#include <assert.h>
 #include <string.h>
 
 void
-vr_config_add_script(struct vr_config *config,
-                     const char *filename)
+vr_config_add_script_file(struct vr_config *config,
+                          const char *filename)
 {
-        struct vr_config_script *script =
-                vr_alloc(sizeof *script + strlen(filename) + 1);
+        struct vr_config_script *script;
+        assert(filename && "|filename| must not be NULL");
+        script = vr_alloc(sizeof *script);
+        script->filename = vr_alloc(strlen(filename) + 1);
```

Comment:
Please use `vr_strdup()` here and in the call below.

Reply:
Done.